build_modules/pubspec.yaml

```diff
@@ -0,0 +1,33 @@
+name: build_modules
+version: 0.0.1
```

Comment:
[nit] can we start this at `0.1.0`? We've started trying to shift over to that for the first release.

Reply:
Done!